admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/DynamicFieldServiceImpl.java

```diff
@@ -261,7 +261,7 @@ public StatusResponseDto deleteDynamicFieldValue(String id) {
 						SchemaErrorCode.DYNAMIC_FIELD_NOT_FOUND_EXCEPTION.getErrorMessage());
 
 			JsonNode valueJson = dynamicField.getValueJson() != null ? objectMapper.readTree(dynamicField.getValueJson()) : null;
-			String code = valueJson.get("code").toString();
+			String code = null!=valueJson ? valueJson.get("code").toString():"";
```

Comment:
throw back exception if valueJson is null

Reply:
Done